index.test.js

```diff
@@ -1,18 +1,80 @@
-//Test for submitting a new task to be added
-// to the to-do list
+// Test for adding a new task to the to-do list
 
-test("Once a user adds an item, the result is shown", () => {
-    const form = document.querySelector("form");
-    const input = document.querySelector("#task"); 
-    input.value = "I'm new!";
-    const addedItem = document.querySelector("#result")
-    const btn = document.querySelector("#btn");
-    btn.click();
-    equal(addedItem.textContent, "I'm new!");
-    //input.value = "";
-    form.reset()
+test("Submitting a new task adds it to the list", () => {
+    // Grab the input tag from html with an id of task
+    const input = document.querySelector("input[id='task']");
+    // Add a value to the input to test it's working
+    input.value = "New task";
+    // Write an if statement to check if the box contains text
+    if(input.value !== "") {
+        console.info(`Pass: Task contains text`)
+    } else {
+        console.error(`Fail: Task doesn't contain text`)
+    }
+    // grab add button element for submitting a task
+    const addBtn = document.querySelector("button[id='add-btn']")
+    // Event click on add button for user submit their task
+    addBtn.addEventListener('click', (event) => {
+    // Stop page from reloading once task is added
+        event.preventDefault();
+        })
+    })
```

Comment:
Hi Milly! 

Good job with the refactoring. Sorry if I am missing something in the code - I am wondering if the test is checking that the task was actually added to the to-do list? At the moment it seems that it is checking if the input value is an empty string (which is a good thing to check) but not sure if it asserts that a new task was added to the output.